feat: support SQL Server in gorm-adapter

xiao-niu-ren opened the pull request:
Fix: https://github.com/casbin/gorm-adapter/issues/191

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/casbin/gorm-adapter?pullRequest=192) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/casbin/gorm-adapter?pullRequest=192) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/casbin/gorm-adapter?pullRequest=192) it.</sub>

yyy1000 reviewed (commented):
I think it could work if it could add sqlserver service in github action.

hsluoyz commented:
@xiao-niu-ren fix CI:

![image](https://user-images.githubusercontent.com/3787410/209925263-bc64837f-9349-47cd-995b-1e661e7cb11f.png)

xiao-niu-ren commented:
> @xiao-niu-ren fix CI:
> 
> ![image](https://user-images.githubusercontent.com/3787410/209925263-bc64837f-9349-47cd-995b-1e661e7cb11f.png)



> nk it could work if it could add sqlserver service in github action.

Yeah, I have added the docker of sqlserver, and changed the port and password of sqlserver, and successfully ran through Actions in my test
